### DISCUSSION TITLE
Increase wake up interval to more than once a day

### DISCUSSION THREAD

**Discussion by adrian-klein** [photoprism/photoprism#3890] [Q&A]
Hi, 
so maybe I'm using Photoprism kinda differently from most other people.

It's installed on my NAS so that I can view my images when I'm not at home.

But I have disabled any upload functionality.
I have disabled all subject recognition or any other AI feature.
For me, it's really just a tool that allows me to have thumbnails in a gallery view, and that can scale down images before sending them over the network.

I usually copy new images from my main computer to the NAS once I've sorted / filtered them, which is like ... once or twice a month.

So for me having it wake up even daily, when I know that no single file has changed, and then scan all files for the following 3 hours, is a bit wasteful.

I'm always pondering whether I should just keep photoprism off most of the time, and only start it when I want to look up some images. But I've tried that, and it's an overhead that has stopped me from looking something just because it would have taken too long.

Is there any chance that the max PHOTOPRISM_WAKEUP_INTERVAL could be raised? 
Why is there a limit anyway, apart from being a sensible value for most usecases?

**Comment by srett** [comment]
Try enabling `PHOTOPRISM_UNSAFE`, then you can set the wakeup interval to 0.
